x/bounty/client/cli/flags.go

```diff
@@ -0,0 +1,11 @@
+package cli
+
+const (
+	FlagEncKeyFile        = "enc-key"
```

Comment:
Both use underscores?
`enc-key` -> `enc_key`
`submit-end"` -> `submit_end"`

Reply:
I think the standard is a `-` for variable name spaces